conanfile.py

```diff
@@ -353,6 +361,7 @@ def library_pattern(library):
                               'WebKit']:
                 self.cpp_info.exelinkflags.append('-framework %s' % framework)
             self.cpp_info.sharedlinkflags = self.cpp_info.exelinkflags
+            self.cpp_info.libs.append('iconv')
```

Comment:
`iconv` became a new dependency in 3.1.3, but only on macOS? 🤔 

If it is required we should probably add [libiconv/1.15](https://github.com/conan-io/conan-center-index/blob/master/recipes/libiconv) as a Conan requirement

Reply:
TBH, I did not put much thought into it. The undefined references popped up on mac for _iconv_open etc. and I just blindly assumed that to be a system library, like the frameworks. But I can try again without and see what causes it...

Reply:
Ok, it looks like wx automatically linked to IConv since I happened to have it on my system. I disabled that and a few other libs with similar behavior. I guess those could eventually be added as conan switches, but for now, OFF is better than sometimes ON.